.github/workflows/binaries.yml

```diff
@@ -59,3 +59,49 @@ jobs:
           push: true
           tags: textile/tableland:latest,textile/tableland:${{ github.ref_name }}
           platforms: linux/amd64, linux/arm64
+  js-release:
+    runs-on: ubuntu-latest
+    if: ${{ success() }}
+    needs: [binaries]
```

Comment:
Using `if` + `needs` allows us to wait until all of the builds have succeeded before we trigger the `js-validator` workflow that will download them.